### ISSUE TITLE
Add JSON parsing exception handling in svzerodsolver application

### ISSUE THREAD

**Issue by ktbolt** [StanfordCBCL/svZeroDPlus#39]
The `svzerodsolver` does not catch any JSON parsing errors but just fails with something like
```
libc++abi.dylib: terminating with uncaught exception of type nlohmann::json_abi_v3_11_2::detail::parse_error: [json.exception.parse_error.101] parse error at line 15, column 5: syntax error while parsing object - unexpected ']'; expected '}'
```

In case a user does not know what `nlohmann::json_abi_v3_11_2::detail::parse_error` is then it would be good to print something a bit more informative like
```
[svzerodsolver] Input JSON file "steadyFlow_RC_R.json" parse error: [json.exception.parse_error.101] parse error at line 15, column 5: syntax error while parsing object - unexpected ']'; expected '}'
```

**Comment by mrp089** [comment]
@ktbolt, also closed with #67?

**Comment by ktbolt** [comment]
@mrp089 Well, I sure did a poor job of PR's and their Issues. This is closed with https://github.com/StanfordCBCL/svZeroDPlus/pull/67. 

However, there is still a problem with error processing in the `svzerodcalibrator` application. I will open an Issue for that.
